src/chapters/unit-testing/mstest.rst

```diff
@@ -0,0 +1,347 @@
+MSTest
+======
+
+MSTest is a framework that provides the methods and assertions
+for writing and executing unit tests in C#. 
+
+.. _csharp-attributes:
+
+.. index:: ! TestClass, ! TestMethod
+
+C# Attributes
+-------------
+
+.. index:: ! attributes
+
+In C#, **attributes** are formalized bits of information about a program. They operate
+somewhere between actual code syntax and a comment on the code. Attributes do not 
+directly affect the code they annotate, but they do supply information to the compiler.
+An attribute is enclosed in square brackets, ``[]``, and placed above the item it decorates. 
+
+To run unit tests using the native tools available in Visual Studio, the attributes **[TestClass]** and 
+**[TestMethod]** are used to indicate that certain classes and methods should be treated as test cases. We 
+describe how to use these in the examples that follow.
+
+Testing Setup
+-------------
+
+.. index:: ! dependency
+
+To test a simple .NET Core console project, we add a test project into the same solution with 
+an MSTest dependency.
```

Comment:
"To test a simple .NET Core console project, we add an MSTest project into the same solution."